tests/components/application-connector/gateway/simple_test.go

```diff
@@ -23,11 +23,13 @@ func (gs *GatewaySuite) TestSimpleCases() {
 				gs.T().Log("Calling", entry.CentralGatewayUrl)
 				res, err := http.Get(entry.CentralGatewayUrl)
 				gs.Nil(err)
```

Comment:
We could make this assert instantly fail the test, which would let us get rid of that nested `if` below

Reply:
Yes, we can do it, the code will be nicer. I suggest to do it in following PR.